modules/fedimint-ln-client/src/lib.rs

```diff
@@ -241,12 +241,11 @@ impl LightningClientExt for Client {
 
         // Wait for the transaction to be accepted by the federation, otherwise the
         // invoice will not be able to be paid
-        //self.transaction_updates(operation_id)
-        //    .await
-        //    .await_tx_accepted(txid)
-        //    .await
-        //    .map_err(|e| anyhow::anyhow!("Offer transaction was not accepted: {:?}",
-        // e))?;
+        self.transaction_updates(operation_id)
+            .await
+            .await_tx_accepted(txid)
+            .await
+            .map_err(|e| anyhow::anyhow!("Offer transaction was not accepted: {:?}", e))?;
```

Comment:
Since you commented it in in the second commit you should be able to remove the while loop in the CLI client

Reply:
Done